sample-app/src/utils/geolocationutils.tsx

```diff
@@ -0,0 +1,56 @@
+import { AnswersActions } from "@yext/answers-headless-react";
+import { SearchIntent } from "@yext/answers-headless";
+
+const defaultGeolocationOptions: PositionOptions = {
```

Comment:
Is `PositionOptions` some built-in type? I don't see it getting defined or imported anywhere.

Reply:
yup, it's a built-in type